How to buy software in an enterprise - mega guide (#9171)
PostHog/posthog.com: contents/founders/how-to-buy-software-enterprise.md

---
date: 2024-08-15

> title: How to buy software as an enterprise

Author: ```suggestion
title: How to buy software in an enterprise company
```

author:
  - charles-cook
  - leon-daly
featuredImage: >-
  https://res.cloudinary.com/dmukukwp6/image/upload/take_my_money_11433edb48.png
featuredImageType: full
tags:
  - Founders
  - Growth
  - Ops & finance
crosspost:
  - Blog
---

Buying software as an enterprise company is a pain in the ass. There's no playbook, everyone has different objectives, and the whole process is littered with surprises. If this is your first time doing this, these are some things you may not realize:

- Buying software at big companies can take an insanely long time. It typically takes six to twelve months to go from deciding you need the software to actually closing the deal.
- Negotiating internally with your team is _much_ more important and difficult than negotiating with the salesperson. 

> - The typical enterprise software buying process involves eight to twelve people – it’s not just you and a finance person who signs it off.

Author: ```suggestion
- The standard enterprise software buying process involves eight to twelve people – it’s not just you and a finance person who signs it off.
```

<NewsletterForm />

### Level 3: Get demos and test the ones I like

![pmf-guide](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/pmf-game-guide/pmf-level-3.png)

**Internal process**

It's time to bring in the right people for the demos. Your manager might be interested, but unless it’s really business critical there's no point bringing someone very senior to this demo.

If you're relatively new and don’t have a huge amount of internal influence yet, bring along a senior teammate who also feels the pain of your problem, as they can help you build a case internally.

Narrow down your choices to two or three and ask to trial them for a specific period of time. Define clear goals for this testing period – don’t just log in and mess around, try and actually solve the problem. Ask yourself if the software fixes the problem you are facing. Avoid being upsold into other products as you can always upgrade later. It’s easier to get a smaller budget approved now and add more stuff in the future than to pitch for big bucks at the beginning. 

> **Managing your salesperson**

Reviewer: ```suggestion
#### Managing your salesperson
```

3. Your salesperson can be of most value during the testing period. Ask them to help set up your trial account, and to set up a Teams/Slack channel with your team to improve the comms during the trial period. _This is where most deals fall through for salespeople_, so they will be very keen to get this part right, especially if you are showing signs of being eager to progress through this stage. Get them to do some grunt work for you.

4. Your salesperson will start to see if their tool actually fits your problem. Remember, it’s okay if it doesn’t, and they will appreciate you just saying, “No, this isn’t for us." Ghosting isn't cool, and you may end up coming back a year later if the tool you go with turns out not to be the right one.

5. The level of support you get during this process is the best you're going to get with this company, so bear in mind the only way is down. Don’t assume massive vendors have good support. 

6. If they don’t have a feature you want, _do not trust their roadmap_ – assume by default that thing you want will never get built.

> **It's a trap!** 

Reviewer: ```suggestion
#### It's a trap!
```

The demo you see is a _very_ polished version of the product and often not reflective of a real customer account. Get your actual data into the product as you might run into performance issues or weird edge cases that a demo account won’t highlight. Only playing around in a sandbox or demo environment is risky business.

### Level 4: It’s time to negotiate

![pmf-guide](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/pmf-game-guide/pmf-level-4.png)

> **Internal process**

Reviewer: ```suggestion
#### Internal process
```

The most important thing here is to know the boundaries that you are allowed to operate in. Does your company do multi-year contracts? Do they require an SLA? Whose contract terms will you use, yours or the vendor’s? If you have a procurement team, you almost certainly will need to let them negotiate the contract at this point as anything you agree without them involved is likely to be thrown away when they get involved. 

Check that the budget you thought existed at the beginning of this process still exists – by now you may be a couple of months in, and things can change. You should also make sure that you have exhausted the chain of seniority for approving the purchase, i.e. there is nobody more senior who could swoop in and block the deal. Now is also a good time to strongly discourage any more ‘looping in’ by other people involved in the process. No, that stakeholder does not need to weigh in, stop it. 

> **Managing your salesperson**

Reviewer: ```suggestion
#### Managing your salesperson
```

Some specific terms to negotiate: 

- Waive the implementation fee (they will onboard you for free anyway)
- No multi-year
- No auto-renewal, as you'll forget to give notice to cancel if you hate it
- Similarly, no 'automatic 10% price increase next year', as you should treat each year as a fresh negotiation
- Ask for payment in installments (finance people love this one weird trick)
- Ask for payment terms (net 90 days, you bastard)
- Make sure that you can terminate the contract under reasonable circumstances

If you tell your salesperson ‘this is my budget’, they will give you a price that is 20% over that anyway, so don’t do this – let them come with a number first. The best way to drive down the price is competing quotes. _Do not bluff here_. Send the actual quote as they will know more about their competitors than you do anyway. Most sales teams operate in quarters, so if your deal comes in at the end of the quarter, you’ll get more flexibility on pricing. 

> **It's a trap!** 

Reviewer: ```suggestion
#### It's a trap!
```

Don’t rush into a decision based on a false sense of urgency led by the salesperson.What typically happens is someone in your company will go, "Wait, why are we rushing so much, let’s pause here" and then the whole deal will stall. Let your internal timeline drive, not the salesperson’s.

### Level 5: How do I actually buy this thing?

![pmf-guide](https://res.cloudinary.com/dmukukwp6/image/upload/v1710055416/posthog.com/contents/images/blog/pmf-game-guide/pmf-level-5.png)

> **Internal process**

Reviewer: ```suggestion
#### Internal process
```

OMG WE'RE NOT DONE YET. 

Your top priority is getting the contract signed, and you should already know who actually does this. Be conscious that execs are signing contracts every day, so be ready to defend yours _again_ even if you think the deal is done and dusted. This is the riskiest bit of the whole process. 

Be prepared for this bit to take four weeks, as buying teams can have huge backlogs of contracts to process, set up on their system, etc. Your company will likely only use something like Docusign, or even a third-party software purchasing service. 

> **Managing your salesperson**

Reviewer: ```suggestion
#### Managing your salesperson
```

Do not let your salesperson nudge the signer – they will annoy them. You need to manage this sensitive bit. They will get desperate because they’re now thinking about closing the deal, hitting quota, and picking out the color of their new Tesla.

> **It's a trap!** 

Reviewer: ```suggestion
#### It's a trap!
```

## We did it! Or did we?

Hooray, you’re done! Except you’re not. You’re still responsible for making sure the software is actually working and useful at your company. The salesperson will tell you that implementation usually takes about X months. Take that number and double it and you’ll be in the right ballpark. Churning software is bad for everyone. Wasted resources, lost data, broken hearts. 

Your salesperson will dump you the microsecond the deal is closed, so you'll now need to make friends with a customer success person of some sort. This is okay because they are product experts and are usually a lot more chill to deal with because they don't have the time pressure that classic salespeople do.

## The 8 final bosses you’ll (probably) need to beat

As you go through this process, you will probably encounter one or more of these people. Here are some quick tips on how to deal with them.

![eight-bosses](https://res.cloudinary.com/dmukukwp6/image/upload/v1723719752/posthog.com/contents/eight-bosses-enterprise-software.png)

> **The procurement person who actually has a personality if you get to know them**
> - When to engage: when you’ve picked your preferred option.
> - What they care about: following the proper internal process,  and getting the software for as cheap as possible.
> - How to win them over: frame your request as asking for their expertise, not telling them what to do. Come prepared with all the notes on the evaluation so far.
> 
> **The engineer who implemented the current solution and is taking it personally that you’re suggesting something else**
> - When to engage: right at the beginning.
> - What they care about: saving face if you replace what they implemented , and not losing any important functionality by changing software.
> - How to win them over: blame the software they implemented, nerd out over technical details.
> 
> **The anxious infosec officer who has GDPR PTSD and hates buying outside the Fortune 50**
> - When to engage: when you’ve picked your preferred option.
> - What they care about: being careful about sending company data to some random software company.
> - How to win them over: send them a SOC 2 or ISO 27001 report, it’s like catnip.
> 
> **The “in the weeds” legal person who loves a redline and hates compromise**
> - When to engage: after you have agreed commercial terms.
> - What they care about: liability, ability to terminate the contract, general legal point scoring.
> - How to win them over: thank them profusely for taking a ‘commercial’ angle and not being overly legalistic.
> 
> **The CEO who thinks we should go with something else because they met the founder at Davos**
> - When to engage: you probably won’t get to engage them, but be prepared to justify your choice at no notice.
> - What they care about: their LinkedIn following.
> - How to win them over:  presentation deck, ten slides, few words, legible on mobile.
> 
> **The CFO who thinks we should use spreadsheets for everything**
> - When to engage - bizarrely, right at the beginning – if you can even reach them.
> - What they care about: the share price.
> - How to win them over: make the economic case, put numbers against time saved, revenue generated etc. 
> 
> **The generic loud person on a totally different team who will only sign in once but give lots of opinions**
> - When to engage: whenever they spring up, you’ll need to be reactive.
> - What they care about: having their voice heard.
> - How to win them over: get on a call, reassure them their feedback will be taken seriously, then throw away your notes
> 
> **The digital transformation guru whose responsibilities are incredibly vague but their solution is always AI**
> - When to engage: whenever really, they are good for helping you build hype internally.
> - What they care about: getting the company to do new stuff.
> - How to win them over: sympathize with them about how resistant to change the organization is.

Reviewer: ```suggestion
**The procurement person who actually has a personality if you get to know them:**

- **When to engage:** When you’ve picked your preferred option.

- **What they care about:** Following the proper internal process, and getting the software for as cheap as possible.

- **How to win them over:** Frame your request as asking for their expertise, not telling them what to do. Come prepared with all the notes on the evaluation so far.

**The engineer who implemented the current solution and is taking it personally that you’re suggesting something else:**

- **When to engage:** Right at the beginning.

- **What they care about:** Saving face if you replace what they implemented, and not losing any important functionality by changing software.

- **How to win them over:** Blame the software they implemented, nerd out over technical details.

**The anxious infosec officer who has GDPR PTSD and hates buying outside the Fortune 50:**

- **When to engage:** when you’ve picked your preferred option.

- **What they care about:** Being careful about sending company data to some random software company.

- **How to win them over:** Send them a SOC 2 or ISO 27001 report, it’s like catnip.

**The “in the weeds” legal person who loves a redline and hates compromise:**

- **When to engage:** After you have agreed commercial terms.

- **What they care about:** Liability, ability to terminate the contract, general legal point scoring.

- **How to win them over:** Thank them profusely for taking a ‘commercial’ angle and not being overly legalistic.

**The CEO who thinks we should go with something else because they met the founder at Davos:**

- **When to engage:** You probably won’t get to engage them, but be prepared to justify your choice at no notice.

- **What they care about:** Their LinkedIn following.

- **How to win them over:** Presentation deck, ten slides, few words, legible on mobile.

**The CFO who thinks we should use spreadsheets for everything:**

- **When to engage:** Bizarrely, right at the beginning – if you can even reach them.

**- What they care about:** The share price.

- **How to win them over:** Make the economic case, put numbers against time saved, revenue generated etc. 

**The generic loud person on a totally different team who will only sign in once but give lots of opinions:**

- **When to engage:** Whenever they spring up, you’ll need to be reactive.

- **What they care about:** Having their voice heard.

- **How to win them over:** Get on a call, reassure them their feedback will be taken seriously, then throw away your notes.

**The digital transformation guru whose responsibilities are incredibly vague, but their solution is always AI:**

- **When to engage:** Whenever really, they are good for helping you build hype internally.

- **What they care about:** Getting the company to do new stuff.

- **How to win them over:** Sympathize with them about how resistant to change the organization is.
```
